get-started/part2.md

```diff
@@ -216,7 +216,7 @@ be something like `--tag=friendlyhello:v0.0.1`.
 
 ## Run the app
 
-Run the app, mapping your machine's port 4000 to the container's published port
+Restart the Docker client. Then run the app, mapping your machine's port 4000 to the container's published port
```

Comment:
Not entirely sure what is meant with "restarting the docker client" (the `docker` cli is not a long-living process, and is started again with each command you run).

@panagiotismavridis could you explain what you meant with "restarting the client"?

Reply:
/cc @ddebroy @StefanScherer

Reply:
Hm, maybe Docker Desktop, but then I only could think of that @paigehargrave already had another container running with machine port 4000 mounted.

Otherwise I don't think this is needed. This tutorial looks good as it is.

Reply:
@panagiotismavridis can you confirm that you had the scenario described by @StefanScherer above? If so, the change may not be necessary. Please let us know what your environment looks like.